module-apps/apps-common/popups/data/PopupRequestParamsBase.cpp

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2017-2021, Mudita Sp. z.o.o. All rights reserved.
+// For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+#include "PopupRequestParamsBase.hpp"
+
+namespace gui
+{
+    PopupRequestParams::PopupRequestParams(gui::popup::ID popupId) : PopupRequestParams(popupId, popup::Disposition{})
```

Comment:
Hmm why move that to separate file ? Now it does not match class name. I assume that this class can't be virtual so renaming it into PopupRequestParamsBase does not make big sense also. I would leave it where it was (PopupRequestParams) and create missing cpp there.

Reply:
I'll try to clean it up than :)